index.bs

```diff
@@ -349,6 +349,218 @@ One example of a feature that should be limited to secure contexts
+This complexity probably would have involved
+behaving differently if the property had the same value on the parent element,
+which is the "code smell" mentioned above that suggests
+that a property likely should not have been inherited.
+</div>
+
+<div class="example">
+Another example is the 'font-size' property, which is inherited.
+It sets the size of the font used for the text in the element,
+and continues to apply to any descendants that don't
+have a declaration setting 'font-size' to a different value.
+
+If the 'font-size' property were not inherited,
+then it would probably have to have an initial value
+that requires walking up the ancestor chain to find the nearest ancestor
+that doesn't have that value.
+This is the "code smell" mentioned above that suggests
+that a property likely should have been inherited.
+</div>
+
+<div class="example">
+A more subtle example is the 'text-decoration' property.
+It could reasonably be described as either an inherited or non-inherited property,
+but the resulting behavior is different in a number of observable ways.
+
+As an inherited property,
+'text-decoration' would draw underlines (or other decorations)
+in the style and position of each piece of text.
+As a non-inherited property, it would draw underlines
+in the style and position of the element that specifies 'text-decoration'.
+These behaviors lead to different results
+when the element that has 'text-decoration' declared
+has descendants that are at a different vertical-alignment (e.g., superscripts),
+in a different color,
+or in a different font size.
+
+The property was designed as not inherited
+because the behavior of having underlines
+have a consistent color, thickness, and position
+was considered the better behavior.
+
+However, in the end, some of the characteristics of the other behavior
+(such as its interaction with 'visibility' or text selection)
+were also seen as desirable, and it ended up being specified in a somewhat hybrid way
+that does involve the "code smell"
+of walking up the ancestor chain looking for text decorations on ancestors.
+</div>
+
+<h3 id="css-computed-value">Choose the computed value type based on how the property should inherit</h3>
+
+Another important choice in the design of every CSS property
+is what its <a>computed value</a> is.
+This is an abstract description of the value that is inherited when the property inherits.
+This is most important for inherited properties, but it does matter for all properties.
+
+Due to a somewhat unfortunate history,
+this is *not* the value that is returned by {{getComputedStyle()}},
+which is actually the <a>resolved value</a>,
+which can be the result of further computation starting from the computed value.
+
+The definition of a CSS property often has interactions with other CSS properties.
+Whether those interactions affect the property's computed value is a design choice,
+and it affects the behavior of inheritance.
```

Comment:
```suggestion
The definition of a CSS property often has interactions with other CSS properties.
Inheritance passes down the [=computed value=], as well,
so whether or not the interaction should affect the value that descendants inherit
is an important factor in whether the interactions are handled in the [=computed value=]
or in a later value stage such as the [=used value=].
```

Reply:
I rewrote this paragraph a different way from what you suggested, but incorporating a bunch of your suggestions.